python/config/config_Rb.cfg

```diff
@@ -28,7 +28,7 @@ PreampGainIndex_4522 = 2 ; Preamp gain index (3rd entry), for camera with serial
 PICAM_DLL = C:\Users\Rb\LabSoftware\CsPyNewGit\CsPyController\python\PythonForPicam\DLLs\Picam.dll
 
 [AAS]
-Camera0DataGroup = camera_data/15102504/stats ; camera_data/`serial number` for blackfly camera
+Camera0DataGroup = camera_data/16483678/stats,camera_data/16483678/stats ; camera_data/`serial number`/stats for blackfly camera
```

Comment:
This is the Hybird branch so Rb configs shouldn't be touched.

Reply:
Anything Rb specific should be discarded from the Hybrid PR. They should be pulled in via the Rb PR, so just discard all of them.